Automattic/jetpack
# Stats: Add stats subscription to My Plan

Fixes #31713 

## Proposed changes:

* Show Stats subscription in My Plan

### Other information:


## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:

* Purchase Stats for your site `https://wordpress.com/checkout/${siteId}/jetpack_stats_monthly`
* Build Jetpack `jetpack build plugins/jetpack`
* Open `/wp-admin/admin.php?page=jetpack#/my-plan`
* Ensure you see Stats / Stats Free listed in My Products

<img width="1084" alt="image" src="https://user-images.githubusercontent.com/1425433/252506177-31e90e0d-922b-46f8-87ff-c2dfa5b6e81d.png">
